release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -122,6 +122,49 @@ Previously, `-9223372036854775808` was allowed to be assigned to an integer in B
 int result = -9223372036854775808; // error: '9223372036854775808' is out of range for 'int'
 ```
 
+- Restricted the typed binding of the query expression outer join to be inferred as a nillable type. Earlier, the typed binding for outer join could be any type descriptor but now it's restricted to be `var` so that it will be inferred as a nillable type.
```

Comment:
`Restricted the typed binding of the query expression outer join to be inferred as a nillable type.`
Shouldn't this be a heading?

Reply:
it is too long for a heading. If we are making a heading out of it, we need to summarize it.

Reply:
Fixed